packages/page/src/Infrastructure/Sulu/Content/PageSitemapProvider.php

```diff
@@ -22,9 +22,9 @@
  */
 class PageSitemapProvider extends ContentSitemapProvider
 {
-    public function __construct(EntityManagerInterface $entityManager, WebspaceManagerInterface $webspaceManager, string $kernelEnvironment, string $contentRichEntityClass, string $routeClass, string $alias)
+    public function __construct(EntityManagerInterface $entityManager, WebspaceManagerInterface $webspaceManager, string $contentRichEntityClass, string $routeClass, string $alias)
     {
-        parent::__construct($entityManager, $webspaceManager, $kernelEnvironment, $contentRichEntityClass, $routeClass, $alias);
+        parent::__construct($entityManager, $webspaceManager, $contentRichEntityClass, $routeClass, $alias);
     }
```

Comment:
This looks like it's the same as the parent constructor. Then it can be removed.